front/pages/api/v1/w/[wId]/assistant/conversations/index.ts

```diff
@@ -135,24 +135,28 @@ async function handler(
         // before returning the conversation along with the message.
         // PostUserMessageWithPubSub returns swiftly since it only waits for the
         // initial message creation event (or error)
-        const messageRes = await postUserMessageWithPubSub(auth, {
-          conversation,
-          content: message.content,
-          mentions: message.mentions,
-          context: {
-            timezone: message.context.timezone,
-            username: message.context.username,
-            fullName: message.context.fullName,
-            email: message.context.email,
-            profilePictureUrl: message.context.profilePictureUrl,
+        const messageRes = await postUserMessageWithPubSub(
+          auth,
+          {
+            conversation,
+            content: message.content,
+            mentions: message.mentions,
+            context: {
+              timezone: message.context.timezone,
+              username: message.context.username,
+              fullName: message.context.fullName,
+              email: message.context.email,
+              profilePictureUrl: message.context.profilePictureUrl,
+            },
           },
-        });
+          { resolveAfterFullGeneration: isSync === true }
+        );
 
         if (messageRes.isErr()) {
           return apiError(req, res, messageRes.error);
         }
 
-        newMessage = messageRes.value;
+        newMessage = messageRes.value.userMessage;
```

Comment:
We likely want the full conversation returned when calling with isSync = true no?

Reply:
If I pass it today, what do I get as a result of using isSync = true?